src/Microsoft.DotNet.Build.Tasks/PackageFiles/versioning.targets

```diff
@@ -234,6 +236,7 @@
   <Target Name="CreateVersionFileDuringBuild" Condition="'$(SkipVersionGeneration)'!='true' AND '$(ShouldCreateVersionFileDuringBuild)'=='true'" DependsOnTargets="CreateOrUpdateCurrentVersionFile">
     <PropertyGroup>
       <BuildNumberMajor>$(GeneratedBuildNumberMajor)</BuildNumberMajor>
```

Comment:
If you want to checkin something, then you can still accomplish this by adding this to dir.props:
```xml
<PropertyGroup>
  <BuildNumberMajor>00001</BuildNumberMajor>
  <BuildNumberMinor>00</BuildNumberMinor>
  <SkipVersionGeneration>true</SkipVersionGeneration>
</PropertyGroup>
```